contrib/scalar/scalar.c

```diff
@@ -19,55 +19,91 @@ static int is_unattended(void) {
 	return git_env_bool("Scalar_UNATTENDED", 0);
 }
 
+/**
```

Comment:
nit: Multi-line comments start with `/*` in the Git codebase.

I wouldn't worry too much for this PR, but let's keep it mind for the upstream version?

Reply:
✅